alacritty/src/window.rs

```diff
@@ -252,11 +255,7 @@ impl Window {
             _ => true,
         };
 
-        let image = image::load_from_memory_with_format(WINDOW_ICON, ImageFormat::Ico)
-            .expect("loading icon")
-            .to_rgba();
-        let (width, height) = image.dimensions();
-        let icon = Icon::from_rgba(image.into_raw(), width, height);
+        let icon = Icon::from_resource(0x101, None);
```

Comment:
What does 0x101 mean? Is there a constant we can use for that?

Reply:
It's the [same constant the icon is declared with in the manifest](https://github.com/alacritty/alacritty/blob/master/extra/windows/windows.rc#L1-L3)

Reply:
could you give it a name and point from where it was taken?